frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -16,7 +16,7 @@
  */
 
 import React from "react"
-import * as _ from "lodash"
+import without from "lodash/without"
```

Comment:
No need to import the whole thing.

Reply:
I didnt test it but it should be the same as create-react-app already includes lodash within it, but I'm agree with you